engine/modelc/src/modelimporter_gltf.cpp

```diff
@@ -899,6 +898,29 @@ static void LoadPrimitives(Scene* scene, Model* model, cgltf_data* gltf_data, cg
             uint32_t size = mesh->m_VertexCount * mesh->m_TexCoords0NumComponents;
             InitSize(mesh->m_TexCoords0, size, size);
         }
+
+        if (prim->indices)
+        {
+            ReadAccessorUint32ToArray(prim->indices, 1, mesh->m_Indices);
+        }
+        else
+        {
+            // for now, we only support triangles
+            if(prim->type == cgltf_primitive_type_triangles)
+            {
+                uint32_t num_vertices = mesh->m_Positions.Size() / 3;
+                mesh->m_Indices.SetCapacity(num_vertices);
+                mesh->m_Indices.SetSize(num_vertices);
+                for (uint32_t i = 0; i < num_vertices; ++i)
+                {
+                    mesh->m_Indices[i] = i;
+                }
+            }
```

Comment:
I guess we could support it at runtime, avoiding the indices data altogether, but I wasn't sure we supported it, so I added the indices here.

The enum supports triangle lists as well, but no such model was in the reference set so, we'll deal with it when we see one.